lld/MinGW/Options.td

```diff
@@ -196,6 +196,7 @@ defm guard_longjmp : B<"guard-longjmp",
   "Do not enable Control Flow Guard long jump hardening">;
 defm error_limit:
   EqLong<"error-limit", "Maximum number of errors to emit before stopping (0 = no limit)">;
+def build_id: F<"build-id">, HelpText<"Generate build ID">;
```

Comment:
Right now, making this a documented option doesn't really make sense, as it is enabled by default and there's no way of switching it off, but if we can make it possible to disable it, then documenting it might be good.